shared/src/businessNameSearch.ts

```diff
@@ -6,8 +6,20 @@ export type NameAvailabilityStatus =
   | "RESTRICTED_ERROR"
   | undefined;
 
-export type NameAvailability = {
+export interface NameAvailabilityResponse {
   status: NameAvailabilityStatus;
   similarNames: string[];
   invalidWord?: string;
+}
+
+export interface NameAvailability extends NameAvailabilityResponse {
+  lastUpdatedTimeStamp: string;
+}
+
+export type BusinessNameSearchResponse = {
```

Comment:
Why the introduction of the third type? I understood your distinction between `NameAvailability` and `NameAvailabilityResponse` that you described but not sure I understand the purpose of `BusinessNameSearchResponse` also